etc/lime-elements.api.md

```diff
@@ -183,14 +183,14 @@ export namespace Components {
     export interface LimelChip {
         "badge"?: string | number;
         "disabled": boolean;
-        "icon": string | Icon;
+        "icon"?: string | Icon;
         "language": Languages;
         "link"?: Omit<Link, 'text'>;
         "readonly": boolean;
         "removable": boolean;
         "selected": boolean;
         "text": string;
-        "type": 'filter';

```

Comment:
These looks like breaking changes

Reply:
Or maybe it's not breaking, it just looks like it could be because it's declared as an interface here, but in reality it's our own component :smile:

Reply:
Yeah, I hade the exact same thought journey, I was just approximately 15 minutes slower than you 😂